etc/kayobe/ansible/smartmon-tools.yml

```diff
@@ -1,9 +1,10 @@
 ---
-- hosts: overcloud
+- name: Install smartmon-tools
```

Comment:
```suggestion
- name: Install and setup smartmon-tools
```

Reply:
https://promova.com/confusing-words/setup-set-up
But I agree with the sentiment

Reply:
Done